server/src/db/migrations/20230209184947_createVotes.cjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("votes", table => {
+        table.bigIncrements("id")
+        table.integer("vote").notNullable()
```

Comment:
we shouldnt have a column name that is the same as the table, as its ambiguous. `value` would be better